.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: olafurpg/setup-scala@v2
+      - uses: olafurpg/setup-scala@v10
       - name: Cache SBT
         uses: coursier/cache-action@v3
       - uses: olafurpg/setup-gpg@v2
```

Comment:
Thanks! While you're in here can you also upgrade setup-gpg to v3? Fine to merge after that.

Reply:
done